.github/linters/.golangci.yml

```diff
@@ -1,40 +1,59 @@
----
-#########################
-#########################
-## Golang Linter rules ##
-#########################
-#########################
-
-# configure golangci-lint
-# see https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml
-run:
-  skip-files: zz_generated*
-  timeout: 3m
-
 issues:
   exclude-rules:
     - path: _test\.go
       linters:
       - dupl
       - gosec
       - goconst
-    
+linters-settings:
+  errcheck:
+    check-type-assertions: true
+  goconst:
+    min-len: 2
+    min-occurrences: 3
+  gocritic:
+    enabled-tags:
+      - diagnostic
+      - experimental
+      - opinionated
+      - performance
+      - style
+    disabled-checks:
+      - unnamedResult
+      - hugeParam
+  nolintlint:
+    require-explanation: true
+    require-specific: true
+
 linters:
-  disable: 
-    - typecheck
+  disable-all: true
```

Comment:
This is from memory but as far as I remember, if you enabled specific linters one by one then you would be disabling all others by default
If that still remains then you don't need this here